packages/block-library/src/embed/youtube-controls.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { PanelBody, ToggleControl, TextControl } from '@wordpress/components';
+import { InspectorControls } from '@wordpress/editor';
+
+export const YouTubeInspectorControls = class extends Component {
+	constructor() {
+		super( ...arguments );
+		const { extraOptions = {} } = this.props.attributes;
+		this.setAttributes = this.setAttributes.bind( this );
```

Comment:
Maybe name this something else, it's a bit confusing to read vs props.setAttributes

Reply:
+1